### DISCUSSION TITLE
求建议，现在48级了但是丁尼严重不足

### DISCUSSION THREAD

**Discussion by qianliyuanqian** [share121/inter-knot#285] [崩坏：星穹铁道]
要不要用体力刷丁尼？感觉好亏，刷素材和圣遗物是否更合适？

**Comment by Elysia-Ego-093** [comment]
可以等两天，马上版本更新后主线会送很多资源，到时候应该足够了，现在还是把体力拿来刷素材好些。
我也是48级的绳匠，天天丁尼见底😂，不过把新的主线和任务做了应该会好很多了。
